src/modules/ttlmonitor/TTLMonitorInstrument.cpp

```diff
@@ -101,73 +104,39 @@ void TTLMonitorInstrument::processMonitorReadouts(void) {
-      continue;
-    }
-
-    if (readout.TDC != 0) {
-      XTRACE(DATA, WAR, "TDC (%u) must be zero", readout.TDC);
-      counters.MonitorErrors++;
-      continue;
-    }
-
-    if (readout.VMM != 0) {
-      XTRACE(DATA, WAR, "VMM (%u) must be zero", readout.VMM);
-      counters.MonitorErrors++;
-      continue;
-    }
 
     uint32_t PixelId = 1;
     XTRACE(DATA, DEB, "Pixel: %u TOF %" PRIu64 "", PixelId, TimeOfFlight);
-    if (UseEveryNEvents == 1) {
       counters.TxBytes +=
-          Serializers[readout.Channel].addEvent(TimeOfFlight, PixelId);
+         Serializers[readout.Channel]->addEvent(TimeOfFlight, PixelId);
       counters.MonitorCounts++;
-      UseEveryNEvents = Settings.TTLMonitorReduceEvents;
-    } else {
-      counters.MonitorIgnored++;
-      UseEveryNEvents--;
-    }
+
   }
 }
 
-/// \todo move into readout/vmm3 instead as this will be common
-void TTLMonitorInstrument::dumpReadoutToFile(
-    const ESSReadout::VMM3Parser::VMM3Data &Data) {
-  VMM3::Readout CurrentReadout;
-  CurrentReadout.PulseTimeHigh = ESSReadoutParser.Packet.HeaderPtr->PulseHigh;
-  CurrentReadout.PulseTimeLow = ESSReadoutParser.Packet.HeaderPtr->PulseLow;
-  CurrentReadout.PrevPulseTimeHigh =
-      ESSReadoutParser.Packet.HeaderPtr->PrevPulseHigh;
-  CurrentReadout.PrevPulseTimeLow =
-      ESSReadoutParser.Packet.HeaderPtr->PrevPulseLow;
-  CurrentReadout.EventTimeHigh = Data.TimeHigh;
-  CurrentReadout.EventTimeLow = Data.TimeLow;
-  CurrentReadout.OutputQueue = ESSReadoutParser.Packet.HeaderPtr->OutputQueue;
-  CurrentReadout.BC = Data.BC;
-  CurrentReadout.OTADC = Data.OTADC;
-  CurrentReadout.GEO = Data.GEO;
-  CurrentReadout.TDC = Data.TDC;
-  CurrentReadout.VMM = Data.VMM;
-  CurrentReadout.Channel = Data.Channel;
-  CurrentReadout.RingId = Data.RingId;
-  CurrentReadout.FENId = Data.FENId;
-
-  DumpFile->push(CurrentReadout);
-}
+// /// \todo move into readout/vmm3 instead as this will be common
+// void TTLMonitorInstrument::dumpReadoutToFile(
+//     const ESSReadout::VMM3Parser::VMM3Data &Data) {
+//   VMM3::Readout CurrentReadout;
+//   CurrentReadout.PulseTimeHigh = ESSReadoutParser.Packet.HeaderPtr->PulseHigh;
+//   CurrentReadout.PulseTimeLow = ESSReadoutParser.Packet.HeaderPtr->PulseLow;
+//   CurrentReadout.PrevPulseTimeHigh =
+//       ESSReadoutParser.Packet.HeaderPtr->PrevPulseHigh;
+//   CurrentReadout.PrevPulseTimeLow =
+//       ESSReadoutParser.Packet.HeaderPtr->PrevPulseLow;
+//   CurrentReadout.EventTimeHigh = Data.TimeHigh;
+//   CurrentReadout.EventTimeLow = Data.TimeLow;
+//   CurrentReadout.OutputQueue = ESSReadoutParser.Packet.HeaderPtr->OutputQueue;
+//   CurrentReadout.BC = Data.BC;
+//   CurrentReadout.OTADC = Data.OTADC;
+//   CurrentReadout.GEO = Data.GEO;
+//   CurrentReadout.TDC = Data.TDC;
+//   CurrentReadout.VMM = Data.VMM;
+//   CurrentReadout.Channel = Data.Channel;
+//   CurrentReadout.RingId = Data.RingId;
+//   CurrentReadout.FENId = Data.FENId;
+//
+//   DumpFile->push(CurrentReadout);
+// }
```

Comment:
Should this be removed too?

Reply:
same reply as before - keep for now, then delete if we find out we don't need it.